web/package.json

```diff
@@ -15,7 +15,9 @@
     "lint": "ng lint",
     "reinstall": "rm -rf node_modules tmp deploy dist && npm install && npm rebuild node-sass",
     "reinstall:latest": "rm -rf node_modules tmp deploy dist && npm install",
-    "test": "npm run lint && npm run build:dev && concurrently -k -s first \"npm start\" \"sleep 15 && npm run e2e:ci\""
+    "test": "npm run lint && npm run build:dev && concurrently -k -s first \"npm start\" \"sleep 15 && npm run e2e:ci\"",
+    "cypress:run": "cypress run",
+    "cypress:open": "cypress open"
```

Comment:
You might need to prepend these commands with `npx` as most people will not have it installed globally but it will be available in `node_modules`

Reply:
okay will add it